Jenkinsfile

```diff
@@ -113,11 +113,18 @@ ansiColor('xterm') {
              nvm use v8.9.1
              NODE_ENV=test npm run build:package widget-space && npm run build:package widget-recents
              CISCOSPARK_CLIENT_ID=C873b64d70536ed26df6d5f81e01dafccbd0a0af2e25323f7f69c7fe46a7be340 SAUCE=true npm test
-             CISCOSPARK_CLIENT_ID=C873b64d70536ed26df6d5f81e01dafccbd0a0af2e25323f7f69c7fe46a7be340 SAUCE=true npm run test:automation -- --suite=oneOnOne
-             CISCOSPARK_CLIENT_ID=C873b64d70536ed26df6d5f81e01dafccbd0a0af2e25323f7f69c7fe46a7be340 SAUCE=true npm run test:automation -- --suite=space
-             CISCOSPARK_CLIENT_ID=C873b64d70536ed26df6d5f81e01dafccbd0a0af2e25323f7f69c7fe46a7be340 SAUCE=true npm run test:automation -- --suite=recents
-             CISCOSPARK_CLIENT_ID=C873b64d70536ed26df6d5f81e01dafccbd0a0af2e25323f7f69c7fe46a7be340 SAUCE=true npm run test:automation -- --suite=multiple
+             CISCOSPARK_CLIENT_ID=C873b64d70536ed26df6d5f81e01dafccbd0a0af2e25323f7f69c7fe46a7be340 SAUCE=true PORT=4567 SAUCE_CONNECT_PORT=5004 BROWSER=chrome npm run test:automation -- --suite=oneOnOne &
+             sleep 60 && CISCOSPARK_CLIENT_ID=C873b64d70536ed26df6d5f81e01dafccbd0a0af2e25323f7f69c7fe46a7be340 SAUCE=true PORT=4568 SAUCE_CONNECT_PORT=5005 BROWSER=chrome npm run test:automation -- --suite=space &
```

Comment:
Is there something we could do instead of sleeping? Are we waiting for something to start before issuing the next command?

Reply:
This is the only workaround I could find to avoid sauce tunnels being started at the same time & crashing and burning on jenkins. AFAIK there's no other commands we could do here instead.

Reply:
Is there a way to signal that the tunnel is setup in the wdio before hook somehow? My concern is that it make take +60 seconds to setup and then the job is failing.

Reply:
I don't *think* so, because I believe the tunnel is started right when the webdriver process is started, before any hooks are executed. I may be wrong on this but I don't believe so.

Reply:
Also, its not that the tunnel needs to be fully setup (it seems), it just crashes when starting at exactly the same time. I've run at least 4-5 test runs that have been successful with this method. Maybe revisit if it becomes a problem?

Reply:
I'm actually more concerned about unnecessarily extending the length of the tests than anything. Having them run it parallel should more than make up for it though! Just wanted to make sure we tried all the options available.

Reply:
Ah, gotcha. Right now, test runs are sitting at ~30 minutes, including build time, so I still think we're saving quite a lot of time even though we have to stagger the build since we doubled the test coverage.

Reply:
![img](https://media0.giphy.com/media/dpnfPvaCIHBrW/giphy.gif?cid=e1bb72ff5a39711d356a4d30777a16c8)